gsy_framework/schema/avro_schemas/launch_simulation_scenario.json

```diff
@@ -0,0 +1,341 @@
+{
+  "name": "ScenarioArea",
+  "type": "record",
+  "namespace": "my.types",
+  "fields": [
+    {"name": "name", "type": "string"},
+    {"name": "type", "type": ["null", "string"], "default":  "undefined"},
```

Comment:
Here the documentatation is also not that clear. Does `"default":  "undefined"` mean that the default value is `None`?

Reply:
If I remember correctly, the `default` key, if specefied, populates the dictionary key if it doesn't exists in the original payload. I'm not sure if we really need it as I have always seen AVRO as a tool to only validate our data and not correct our mistakes. What do you think?

Reply:
It is exactly as @mrsaemir describes, it populates the default value with None. We unfortunately need this because these classes are not only validators, but serializers too (essentially they intend to also "correct our mistakes" to an extent before serializing and deserializing the dict).
The reason for its existence in the first place is that the "type" key in the area representation is optional in our model. Non-existence means that the area is of type "Area", which is equal to "type": None. I tried to use "Area" as the default value in a former attempt but was unsuccessful. I'll try it again now and maybe it will work out.

Reply:
At the time of writing this comment, I wasn't fully convinced to act as a serializer, too. So, I'm all good with default values now. Thanks